Week2/homework-midy/oddOnesOut.js

```diff
@@ -0,0 +1,9 @@
+const myNumbers = [1, 2, 3, 4, 5, 6];
+
+function doubleEvenQuicker(numbers) {
+  const evens = numbers.filter(num => num % 2 === 0);
+  const doubledEvens = evens.map(num => num * 2);
+  return doubledEvens;
```

Comment:
Again here. `doubleEvens` variable declaration is redundant.

Reply:
Yeah I could've returned it while mapping it.